### PR TITLE
chm-404 UNDB Party profile (Country) view page

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrage UNDP Party page from chm project

### Designs
same UI as chm project

### Testing instructions
search-> UNDP Party Profile
image

### CODE REVIEW

File: app/views/forms/view/chm/undb-party.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      
+            <div v-if="document.description">             
+                <label>{{ t("description") }} </label>                            
+                <ng v-vue-ng:km-value-ml  :value="document.description" :locales="locale" html></ng> 
+            </div>
+      
+            <div v-if="document.descriptionNative">         
+                <label>{{ t("descriptionNativeLanguage") }} </label>   
+                <div class="km-value">
+                    <span>{{document.descriptionNative}}<br/></span>
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/undb-party.vue:19]: do we need '<br/>' here ?

File: app/views/forms/view/chm/undb-party.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                </div>
+            </div>
+      
+            <div v-if="document.documents">             
+                <label>{{ t("documents") }} </label>   
+                <div class="km-value">
+                    <div v-for="item in document.documents">
+                        <a  :href="item.url" target="_blank">{{item.name||item.url}}</a>      
+                    </div>
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/undb-party.vue:37]: can we use <ng v-vue-ng:km-link-list here ?

File: app/views/forms/view/chm/undb-party.vue (hunk 1 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/undb-party.vue]: remove it

File: app/views/forms/view/chm/undb-party.vue (hunk 2 above)
Comment (liyuanmontreal) [reply app/views/forms/view/chm/undb-party.vue]: changed to km-link-list

**Review by tahmeedshah — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**